### ISSUE TITLE
Convert C{NoneType} and L{NoneType} to L{None} in all docstrings

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8493]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8493|
|Type|enhancement|
|Created|2016-06-22 15:53:22Z|
|Branch|https://github.com/twisted/twisted/tree/8493-rodrigc-none|

As a followup to [ticket:8456], we need to convert C{NoneType} and L{NoneType} to L{None} in all docstrings

<details><summary>Searchable metadata</summary>

```
trac-id__8493 8493
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__None None
branch__8493_rodrigc_none 8493-rodrigc-none
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1466610802828776 1466610802828776
changetime__1466718979463555 1466718979463555
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_1"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/206

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_2"></a>|@adiroiban set owner to @rodrigc|
|-|-|

Changes looks good.

I have triggered the buildbot tests.

Travis-CI tests are only for screening. Before merging something we need to have all tests run on Buildbot.... I am working to auto-trigger them and to report status in PR.

Once they are green, please merge.

Thanks!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set status to `closed`|
|-|-|

In changeset f77c34b33e50e2e43e75d19d2f5c62319782840b
```
#!CommitTicketReference repository="" revision="f77c34b33e50e2e43e75d19d2f5c62319782840b"
Merge 8493-rodrigc-none: In docstrings, change L{NoneType} and C{NoneType} to L{None}

Author: rodrigc
Reviewer: adiroiban
Fixes: #8493
```
